mvn-defaults/pom.xml

```diff
@@ -41,8 +41,6 @@
 
         <PROG>nb5</PROG>
         <maven.plugin.validation>VERBOSE</maven.plugin.validation>
-
-        <jacoco.version>0.8.12</jacoco.version>
```

Comment:
This stemmed from https://github.com/nosqlbench/nosqlbench/pull/1942#discussion_r1597161193